doc/source/release/1.15.0-notes.rst

```diff
@@ -64,16 +172,131 @@ New features
 
 `scipy.special` improvements
 ============================
-
+- The factorial functions ``special.{factorial,factorial2,factorialk}`` now
+  offer an extension to the complex domain by passing the kwarg
+  ``extend='complex'``. This is opt-in because it changes the values for
+  negative inputs (which by default return 0), as well as for some integers
+  (in the case of ``factorial2`` and ``factorialk``; for more details,
+  check the respective docstrings).
+- `scipy.special.zeta` now defines the Riemann zeta function on the complex
+  plane.
+- `scipy.special.softplus` computes the softplus function
+- The spherical Bessel functions (`scipy.special.spherical_jn`,
+  `scipy.special.spherical_yn`, `scipy.special.spherical_in`, and
+  `scipy.special.spherical_kn`) now support negative arguments with real dtype.
+- `scipy.special.logsumexp` now preserves precision when one element of the
+  sum has magnitude much bigger than the rest.
+- The accuracy of several functions has been improved:
+
+  - `scipy.special.ncfdtr` and `scipy.special.nctdtr` have been improved
+    throughout the domain.
+  - `scipy.special.hyperu` is improved for the case of ``b=1``, small ``x``,
+    and small ``a``.
+  - `scipy.special.logit` is improved near the argument ``p=0.5``.
+  - `scipy.special.rel_entr` is improved when ``x/y`` overflows, underflows,
+    or is close to ``1``.
 
 `scipy.stats` improvements
 ==========================
-
+- A new probability distribution infrastructure has been added for the
+  implementation of univariate, continuous distributions:
+
+  - Speed, accuracy, and memory advantages of the new infastructure are
+    detailed in
```

Comment:
mostly note to self--this is the only "incomplete" sentence I noticed last night, probably because I transcribed the wiki release notes a bit early

Reply:
I think I left a blank to add a link to a tutorial when it merges.